tools/msvs/msi/nodemsi.wixproj

```diff
@@ -72,7 +72,10 @@
     </HeatDirectory>
   </Target>
   <PropertyGroup>
-    <PostBuildEvent>move "!(TargetPath)" "$(TargetDir)\$(TargetFileName)"
-move "!(TargetPdbPath)" "$(TargetDir)\$(TargetPdbName)"</PostBuildEvent>
+    <PostBuildEvent>
+      "$(WixToolPath)\torch.exe" -t language "$(TargetDir)en-US\$(TargetFileName)" "$(TargetDir)de-de\$(TargetFileName)" -out "$(TargetDir)transforms\de-de.mst"
+      cscript.exe WiSubStg.vbs "$(TargetDir)en-US\$(TargetFileName)" "$(TargetDir)transforms\de-de.mst" 1031
+      cscript.exe WiLangId.vbs "$(TargetDir)en-US\$(TargetFileName)" Package 1033,1031
+    </PostBuildEvent>
```

Comment:
This sets the value of the installer's `Languages` attribute to all the languages, the installer now supports (both en-us and de-de)